src/components/editor/property/LiteralTypeLabel.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+const LiteralTypeLabel = ({ propertyTemplate }) => {
+  const labelText = (propertyTemplate) => {
+    const typeLabelLookup = {
+      "http://www.w3.org/2001/XMLSchema/integer": "integer",
+      "http://www.w3.org/2001/XMLSchema/dateTime": "date time",
+      "http://www.w3.org/2001/XMLSchema/dateTimeStamp":
```

Comment:
This URI doesn't resolve. Is it a typo?

Reply:
According to https://www.w3.org/TR/rdf11-concepts/#section-Datatypes, it should be `http://www.w3.org/2001/XMLSchema#<DATETYPE>`, e.g., `http://www.w3.org/2001/XMLSchema#integer`.

Can you make this change here and everywhere else in the codebase?

Reply:
According to https://www.w3.org/TR/rdf11-concepts/#section-Datatypes, should it be `http://www.w3.org/2001/XMLSchema#xsd:integer` instead of `http://www.w3.org/2001/XMLSchema#integer` ?

Reply:
I've poked around the internets. It is `http://www.w3.org/2001/XMLSchema#integer`.

Reply:
ok, changed